<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgDummies</class>
 <widget class="QDialog" name="CQDlgDummies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dummy Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Visual properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Object size [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqSize">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="qqColor">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>25</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Dummy - dummy linking</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linked dummy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Link type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qqLinkedDummyCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>50</number>
    </property>
   </widget>
   <widget class="QComboBox" name="qqLinkTypeCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>45</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>50</number>
    </property>
   </widget>
   <widget class="QToolButton" name="qqEditEngine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>65</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Engine properties</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="qqTrajectoryGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Trajectory</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqfollowParentOrientation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>401</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Follow parent path orientation (only indirect parent)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqFollow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>401</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Follow parent path (only direct parent)</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Distance offsets</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="qqOffset">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Offset [m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>25</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy incr. [m]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqIncrement">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>25</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="qqFixed">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>75</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Fixed on path</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqFree">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>75</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Free on path</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="qqApplyMainProperties">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>147</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>147</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>147</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Apply to selection</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqLinkedDummyCombo</tabstop>
  <tabstop>qqLinkTypeCombo</tabstop>
  <tabstop>qqEditEngine</tabstop>
  <tabstop>qqSize</tabstop>
  <tabstop>qqColor</tabstop>
  <tabstop>qqApplyMainProperties</tabstop>
  <tabstop>qqfollowParentOrientation</tabstop>
  <tabstop>qqFollow</tabstop>
  <tabstop>qqFree</tabstop>
  <tabstop>qqFixed</tabstop>
  <tabstop>qqOffset</tabstop>
  <tabstop>qqIncrement</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
